.gitignore

```diff
@@ -1,3 +1,11 @@
+# Generated HTML and other static files
 build/
+# Commonly ignored Node.js files
 node_modules/
 npm-debug.log
+.npm/
+
+# OSX system files, the bane of our existence
```

Comment:
I'm also used to including these in every project. I never remember to add these globally, so I like to not "require" others to do it either.